build.sbt

```diff
@@ -319,6 +319,27 @@ lazy val `play-ahc-ws-standalone` = project
     `shaded`
   ).disablePlugins(sbtassembly.AssemblyPlugin)
 
+//---------------------------------------------------------------
+// Akka Http implementation of WS
+//---------------------------------------------------------------
+
+lazy val `play-akka-http-ws-standalone` = project
```

Comment:
I think it might be worth naming `play-akka-http-ws-experimental`, since this is still in development and we could possibly change APIs. WDYT @marcospereira?

Reply:
Agree.

It would be consistent with the way we changed from Netty to Akka HTTP on the server backend. It is also easier to set users expectations.